frontend/src/components/layout/navigation/Navigation.tsx

```diff
@@ -0,0 +1,98 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useLocalization } from "@fluent/react";
+import styles from "./Navigation.module.scss";
+import { SignUpButton } from "./SignUpButton";
+import { SignInButton } from "./SignInButton";
+import { ProfileData } from "../../../hooks/api/profile";
+import { UpgradeButton } from "./UpgradeButton";
+import { WhatsNewMenu } from "./whatsnew/WhatsNewMenu";
+import { UserMenu } from "./UserMenu";
+import { AppPicker } from "./AppPicker";
+import { MenuToggle } from "./MenuToggle";
+
+export type Props = {
+  theme: "free" | "premium";
+  hasPremium: boolean;
+  mobileMenuExpanded: boolean | undefined;
+  handleToggle: CallableFunction;
+  profile: ProfileData | undefined;
+  isLoggedIn: boolean;
+};
+/** Switch between the different pages of the Relay website. */
+export const Navigation = (props: Props) => {
+  const { l10n } = useLocalization();
+  const router = useRouter();
+  const {
+    theme,
+    mobileMenuExpanded,
+    handleToggle,
+    profile,
+    hasPremium,
+    isLoggedIn,
+  } = props;
+  const homePath = isLoggedIn ? "/accounts/profile" : "/";
+
+  const ToggleButton = () => (
+    <button
+      className={styles["menu-toggle"]}
+      aria-expanded={mobileMenuExpanded}
+      aria-label={l10n.getString("menu-toggle")}
+      aria-controls="mobile-menu"
```

Comment:
Btw, if you do want to link two elements together, and want to make sure that they have the same ID, but still unique IDs in case the component is present twice on the same page, react-aria also has [`useId`](https://react-spectrum.adobe.com/react-aria/useId.html) (and React 18 will even provide that natively, but we're still on 17).

Reply:
In `Navigation.tsx` the `ul` under the `nav` has an id of `mobile-menu`

Reply:
Removed